oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/dto/security/login/LoginMessage.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hp.ov.sdk.dto.security.login;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+public class LoginMessage implements Serializable {
```

Comment:
Several Login DTOs share some attributes like type, created, modified, URI, etc. You could create a "base" Login DTO containing these fields and use inheritance to specify other attributes as necessary (similar to the BaseModelResource object and its derived classes).